lm-agent/tests/conftest.py

```diff
@@ -1300,3 +1326,70 @@ def olicense_output_no_licenses():
           ftire_adams;         	FreeFloating;	1;	2023-02-28 23:59:00;
         """
     )
+
+
+@fixture
+def dsls_output():
+    """
+    Some DSLS output to parse.
+    """
+    return dedent(
+        """\
```

Comment:
All the other fixtures are escaped like this, if I use `strip()` on this one, it would be good to apply to the others too to keep consistency.
Do you think it's worth it?

Reply:
Nope, not worth it!